core/src/main/scala/org/apache/pekko/persistence/cassandra/EventsByTagMigration.scala

```diff
@@ -163,7 +163,7 @@ class EventsByTagMigration(
    * Migrates the entire `messages` table to the the new `tag_views` table.
    *
    * Before running this you must run the migration of the `all_persistence_ids`
-   * table as described in https://doc.akka.io/docs/akka-persistence-cassandra/current/migrations.html#all-persistenceIds-query
+   * table as described in https://pekko.apache.org/docs/pekko-persistence-cassandra/current/migrations.html#all-persistenceIds-query
```

Comment:
this link does not work - could we revert to the working link that we had before?

Reply:
makes sense

Reply:
this comment applies to an old version of akka-persistence-grpc - so another option is to remove this paragraph - the one starting 'Before running this'.

Reply:
Those migrations are pretty old. Since we’re expecting users to first upgrade to the latest version of akka-persistence-cassandra we could remove those migrations completely.